packages/medusa/src/api/routes/admin/variants/get-inventory.ts

```diff
@@ -96,7 +96,7 @@ export default async (req, res) => {
   const [rawChannels] = await channelService.listAndCount({})
   const channels: SalesChannelDTO[] = await Promise.all(
     rawChannels.map(async (channel) => {
-      const locations = await channelLocationService.listLocations(channel.id)
+      const locations = await channelLocationService.listLocationIds(channel.id)
```

Comment:
suggestion: Since the method is named `listLocationIds` I would expect the result to be a collection of ids but here it is named locations so I could think that it return some locations instead. I suggest to rename the const to `locationIds` and then assign it to locations in the returned object. We can also avoid a copy of the object. wdyt of the following

```ts
     channel["locations"] = await channelLocationService.listLocationIds(channel.id)
     return channel
```

Reply:
Same suggestion everywhere we list the ids and store them in a locations const below